web/cypress/e2e/onboarding.spec.ts

```diff
@@ -1,22 +0,0 @@
-import { onOnboardingPage } from "@businessnjgovnavigator/cypress/support/page_objects/onboardingPage";
-
-describe("Onboarding [feature] [all] [group2]", () => {
-  beforeEach(() => {
-    cy.loginByCognitoApi();
-  });
-
-  it("open and closes contextual info panel on onboarding screens", () => {
```

Comment:
Are we planning on adding this test back in? The other deleted tests look like they're incorporated in the new onboarding tests

Reply:
The short answer is no, we have unit tests that cover small features like the contextual panel. While we build out other cypress tests, existing test well get deleted.

We decide to limit the scope for onboarding - https://www.figma.com/file/HHxMhhiQRN0FkzbJKiTWAu/BFS-User-Flows?type=design&node-id=1232%3A26&mode=design&t=nLJ9KgupOuGnKpOs-1